bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/internal/ChannelCommandDescriptionProvider.java

```diff
@@ -90,7 +90,7 @@ public ChannelCommandDescriptionProvider(final @Reference ItemChannelLinkRegistr
                 CommandDescription dynamicCommandDescription = dynamicCommandDescriptionProvider
                         .getCommandDescription(channel, originalCommandDescription, locale);
                 if (dynamicCommandDescription != null) {
-                    if (dynamicCommandDescription == originalCommandDescription) {
```

Comment:
Can we revert this? There was a reason to use the comparison against the reference here: https://github.com/openhab/openhab-core/pull/1817